### PR TITLE
Feature #128300 feat: TJreports API for SMC

### PR DESCRIPTION
**Pull request by monikaatvowel**
Fix language constant issue

### CODE REVIEW

File: src/reports/filters.php [hunk 1]
```diff
@@ -33,7 +33,10 @@ public function get()
 		{
 			ApiError::raiseError(400, JText::_('PLG_API_REPORTS_REPORT_NAME_MISSSING'), 'APIValidationException');
 		}
-		
+
+		$lang = JFactory::getLanguage();
+		$lang->load('', JPATH_ADMINISTRATOR, 'en-GB', true);
```
Comment (manojLondhe) [inline src/reports/filters.php:38]: I think this needs to be `$lang->load('plg_api_reports', JPATH_ADMINISTRATOR, 'en-GB', true);` @monikaatvowel

Comment (manojLondhe) [reply src/reports/filters.php:38]: refer https://github.com/techjoomla/plg_api_easysocial/blob/master/src/easysocial.php#L34

Comment (monikaatvowel) [reply src/reports/filters.php:38]: This loads Joomla default language file only

Comment (thite-amol) [reply src/reports/filters.php:38]: @monikaatvowel This will be required for all API plugins. Must be loaded with the com_api component.

cc
@manojLondhe

**Pull request merged**